docs/application/native/guides/machine-learning/machine-learning-training.md

```diff
@@ -832,7 +875,4 @@ You can use the trained model with pipeline API as follows:
 ## Related information
 
 - Dependencies
-  - Tizen 6.0 and Higher for Mobile
-  - Tizen 6.0 and Higher for Wearable
-  - Tizen 6.0 and Higher for TV
-  - Tizen 6.0 and Higher for IoT
+  - Tizen 6.0 and Higher
```

Comment:
```suggestion
  - Tizen 6.0 and Higher for Mobile
  - Tizen 6.0 and Higher for Wearable
  - Tizen 6.0 and Higher for TV
  - Tizen 6.0 and Higher for IoT
```
I think this format is more appropriate for Related information section in native articles.
Ref: https://github.com/Samsung/tizen-docs/blob/master/reviewguide/review_points_guide.md#dependencies

Reply:
thank you.